app/adapters/external/cache/infinite_buying_adapter.py

```
@@ -181,7 +193,9 @@ async def save_state(self, market: str, state: InfiniteBuyingState) -> bool:
             key = self._get_state_key(market)
             data = self._serialize_state(state)
             result = await self.cache.set(key, data, ttl=86400)  # 24시간 TTL
-            logger.debug(f"무한매수법 상태 저장 완료 - market: {market}, phase: {state.phase.value}")
+            logger.debug(
+                f"무한매수법 상태 저장 완료 - market: {market}, phase: {state.phase.value}"
+            )
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)